owncloudTestUtil/src/main/java/com/owncloud/android/testutil/OCFile.kt

```diff
@@ -34,3 +34,18 @@ val OC_FILE = OCFile(
     mimeType = "DIR",
     length = 123123123
 )
+
+val OC_FILE = OCFile(
+    id = 124,
+    parentId = 122,
+    remotePath = "/Photos/image.jpt",
+    owner = OC_ACCOUNT_NAME,
+    permissions = "RDNVCK",
+    remoteId = "00000003oci9p7er2how",
+    privateLink = "http://server.url/f/4",
+    creationTimestamp = 0,
+    modifiedTimestamp = 1593510589000,
+    etag = "5efb0c13c688f",
+    mimeType = "image/jpeg",
+    length = 3000000
+)
```

Comment:
I like having a `OC_FILE` and a `OC_FOLDER` 💯 

BTW, add a newline at end of file.

Reply:
done